### ISSUE TITLE
GIVGardens - Collateral page 

### ISSUE THREAD

**Issue by DonnaOftadeh** [1Hive/gardens#303]
Great to see it is updated to deposit manage but still it needs improvements:
![image](https://user-images.githubusercontent.com/53400757/130090042-7800d359-a67b-4826-ab81-d7afca0425bd.png)
1. I think it is better to combine the balance and deposit and how I put tokens on various projects would worth having it all together in one place.
2. I think this can be visual without description: with a donut chart (locked and unlocked amount)

**Comment by 0xmiel** [comment]
Agree, we need to do a redesign here. Something to consider for a v2
